ig.ini

```diff
@@ -1,3 +1,3 @@
 [IG]
-ig = fsh-generated/resources/ImplementationGuide-hl7.fhir.us.smarthealthcards-vaccination.json
+ig = fsh-generated/resources/ImplementationGuide-hl7.fhir.uv.smarthealthcards-vaccination.json
```

Comment:
This is surprising to me -- up to this point our modeling decisions (e.g., CVX codes, LOINC) and participation have been US-centric. Are we saying 1) we expect this to change, or 2) we want to make an argument that US-centric choices may be well suited for universal adoption?